### ISSUE TITLE
TextInputGroup - search input group examples should use unique aria-labeling

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7289]
**Describe the enhancement or change**
Since a11y docs will likely include verbiage about using unique aria labeling if there's more than 1 search input on a page, core examples should be updated to provide that unique labeling on inputs/utility buttons.

**Any other information?**

**Comment by Mash707** [comment]
@thatblindgeye I would like to work on this. One thing I need to clarify, we need to add the aria-label to the [`TextInputGroupMain`](https://github.com/patternfly/patternfly-react/blob/4ea82f7af5e42834f2a906ec5c7cc1912466ca2b/packages/react-core/src/components/TextInputGroup/TextInputGroupMain.tsx#L37) component since [`TextInputGroup`](https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/TextInputGroup/TextInputGroup.tsx) does not have a aria-label prop

**Comment by thatblindgeye** [comment]
@Mash707 correct! Thanks for catching that; opened this speaking on the component as a whole but yeah this is meant for TextinputGroupMain
